screenshots/src/main/java/com/appunite/loudius/screenshots/theme/Theme.kt

```diff
@@ -69,13 +69,14 @@ fun LoudiusTheme(
             val context = LocalContext.current
             if (darkTheme) dynamicDarkColorScheme(context) else dynamicLightColorScheme(context)
         }
+
         darkTheme -> DarkColorScheme
         else -> LightColorScheme
     }
     val view = LocalView.current
     if (!view.isInEditMode) {
         SideEffect {
-            (view.context as Activity).window.statusBarColor = colorScheme.primary.toArgb()
+            (view.context as? Activity)?.window?.statusBarColor = colorScheme.primary.toArgb()
```

Comment:
Question: Why do we need to make the safe call?

Reply:
Good question 👌 
The problem (actually it's more like a feature) is that paparazzi not launching views in edit mode. Because of that, the build is failing - Context is type `BridgeContext` instead of `Activity`.

Reply:
Nice, thanks for the answer 😼